elixir/apps/api/lib/api/router.ex

```diff
@@ -52,7 +52,6 @@ defmodule API.Router do
 
     resources "/actors", ActorController, except: [:new, :edit] do
       resources "/identities", IdentityController, except: [:new, :edit, :update]
-      post "/providers/:provider_id/identities/", IdentityController, :create
```

Comment:
Ah, this was actually my mistake.  I made this change toward the end and forgot to come back to the router and make the proper change.  The route being deleted here is actually intended and the `create` route created by the `resources` function was meant to be removed.  So, this should actually be:
```
resources "/identities", IdentityController, except: [:new, :edit, :create, :update]
post "/providers/:provider_id/identities/", IdentityController, :create
```

Reply:
Ok, that was the part I wasn't sure about. I updated the code and tests, it should be good now